### PR TITLE
Switching from rollup -> webpack and implementing CI deployments.

### PR DESCRIPTION
**Pull request by joeldenning**
Rollup doesn't have an equivalent for webpack-dev-server, so I switched to webpack so I could implement the `start` script that watches your files locally and lets you override this module in a deployed environment.

Additionally, webpack is the industry standard and the tool I'd recommend to others for bundling, so switching it here makes it so that people looking at this repo for reference won't be confused by it.

Also, I added to the travis.yml so that https://spa-modules.nyc3.digitaloceanspaces.com/import-map.json is updated whenever we merge into master -- continuous deployments.

### CODE REVIEW

**Comment by brandones** [comment]
LGTM

**Comment by joeldenning** [comment]
@fatmali 👍 yeah I'll change it to a Travis env variable

**Comment by angshu** [comment]
LGTM as well.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+`@openmrs/root-config`
+
+A javascript module for setting up and configuring your SPA. See [openmrs-rfc-frontend](https://github.com/openmrs/openmrs-rfc-frontend)
+for what this project is and how it works.
+
+## Local development
+
+```sh
+npm install
+npm start <port-number> # e.g., npm start 8080
```
Comment (angshu) [inline README.md]: would this be 
> npm start -- --port <port numer>

update document for HTTPs as well?
> npm start -- --https --port <port number>
> npm start -- --https --key ./path-to-key --cert ./path-to-public --port <port number>

Comment (joeldenning) [reply README.md]: see https://github.com/openmrs/openmrs-esm-devtools/pull/3.

Also, the documentation will all be moving to the wiki this week.

**Pull request merged**